<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(155, 194, 231);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(155, 194, 231);
	border-radius: 13px;
	padding: 10px;
}
QLineEdit:hover{
	border: 2px solid rgb(255, 207, 0);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 165, 0);
	color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
    <property name="placeholderText">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(155, 194, 231);
	border-radius: 13px;
	padding: 10px;
}
QLineEdit:hover{
	border: 2px solid rgb(255, 207, 0);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 165, 0);
	color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(155, 194, 231);
	border-radius: 13px;
	padding: 10px;
}
QLineEdit:hover{
	border: 2px solid rgb(255, 207, 0);
}

QLineEdit:focus{
	border: 2px solid rgb(255, 165, 0);
	color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="maxLength">
     <number>35</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>SENHA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	color: rgb(255, 255, 255);
	background-color: rgb(176, 176, 176);
	border: 2px solid rgb(166, 166, 166); 
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgb(166, 166, 166);
	border: 2px solid rgb(138, 138, 138); 
}

QPushButton:pressed {
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24); 
}</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CADASTRO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {	
	color: rgb(255, 255, 255);
	background-color: rgb(176, 176, 176);
	border: 2px solid rgb(166, 166, 166); 
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgb(166, 166, 166);
	border: 2px solid rgb(138, 138, 138); 
}

QPushButton:pressed {
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24); 
}</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/astronalta lua.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
